Cargo.toml

```diff
@@ -68,3 +68,4 @@ rustc-demangle = { opt-level = 3 }
 [patch.crates-io]
 parquet2 = { version = "0.13", optional = true, git = "https://github.com/datafuse-extras/parquet2", branch = "parquet2-0.13-patch2" }
 chrono = { git = "https://github.com/datafuse-extras/chrono", rev = "279f590" }
+goldenfile = { git = "https://github.com/psiace/rust-goldenfile", rev = "16c5783" }
```

Comment:
Please add a comment with links to https://github.com/calder/rust-goldenfile/pull/7 so that we can switch back to upstream later.